src/create-menu/createMenu.js

```diff
@@ -72,6 +95,7 @@ function createMenu(
     {
         hasDefaultQuickReply = true,
         hasWppQuickReply = true,
+        hasWppListMenu = true,
```

Comment:
I think that the text menu is more used in whatsapp because the most developers don't know about list menu. It was released recently and had not a good divulgation. I think that this menu shoud be default for this case because is more friendly for users and is more indicate for chatbots.

Reply:
About this topic. Have a study about ux with both components, list and text number menu and to list you have 3 clicks to get a response and it's specific for some scene and with text 2 clicks, and the option appears before. I think  the text default  better because this and because is most general option to work. That not turn unavailable the dev change after clone this repository,  if on the client context the default  lost is better, is just a default  value, changes with the project.

Reply:
Nice point. I Changed the text menu for default again.